### PR TITLE
Upgrade to 1.4.5

### PR DESCRIPTION
**Pull request by nbeguier**
Add travis support and fix Health() class description

### CODE REVIEW

**Comment by nbeguier** [comment]
Then, I could make a PR of the 1.4.6 version of the server

**Review by vdemonchy — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # CASSH
 
+[![Build Status](https://travis-ci.org/nbeguier/cassh.svg?branch=master)](https://travis-ci.org/nbeguier/cassh)
```
Comment (aladjadj) [inline README.md:3]: Replace nbeguier by Leboncoin

File: cassh [hunk 2]
```diff
@@ -124,8 +124,7 @@ class CASSH(object):
         else:
             if prefix is None:
```
Comment (aladjadj) [inline cassh:125]: Replace by 
else if : 
else:

**Comment by aladjadj** [comment]
I'm not ok to merge this PR :)

**Comment by nbeguier** [comment]
You disagree by default or you have something in mind ?
I will make another commit to hide the travis insert in the readme

**Comment by aladjadj** [comment]
I'm on my mobile and my comments are pending ... Sorry.

**Review by aladjadj — COMMENTED** [review]
My comments

**Pull request merged**